server/src/main/java/io/spine/server/event/EventStore.java

```diff
@@ -45,29 +51,16 @@
  */
 public class EventStore implements AutoCloseable {
 
+    private static final String DIFFERENT_TENANT_ERROR_TEMPLATE =
+            "Events belonging to different tenants (%s) and (%s) cannot be stored in a single operation.";
```

Comment:
`Events, that target different tenants, cannot be stored in a single operation. Observed tenants are: (%s)`  <-- here you list _all_ the tenants that were there.